File: Results/DER/Translation/translation_output/w_test/codellama/codenet/python/java/p02817.java
<s> [INST] <<SYS>>
You are an expert java programmer and assistant
<</SYS>>

Translate the following python code to java and enclose your solution inside ```java```.
A sample test case is provided below:

Test input:
oder atc

Expected output:
atcoder


```
S, T = input().split()

print(T + S)
```
[/INST]
```java
import java.util.Scanner;

public class Solution {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input = scanner.nextLine();
        String[] words = input.split(" ");
        String firstWord = words[0];
        String secondWord = words[1];
        System.out.println(secondWord + firstWord);
    }
}
```
This code uses the `Scanner` class to read input from the user, and the `split()` method to split the input into two words. It then prints the second word followed by the first word.

Here's an example of how to run this code:
```
$ java Solution
oder atc
atcoder
```</s>

File: Results/DER/Translation/translation_output/w_test/codellama/codenet/java/python/p02817.py
<s> [INST] <<SYS>>
You are an expert python programmer and assistant
<</SYS>>

Translate the following java code to python and enclose your solution inside ```python```.
A sample test case is provided below:

Test input:
oder atc

Expected output:
atcoder


```
import java.util.Scanner;

public class p02817 {

	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		String string = in.next();
		String tString = in.next();
		System.out.println(tString + string);
		in.close();
	}
}
```
[/INST]
```
import sys

string = sys.stdin.readline().strip()
tString = sys.stdin.readline().strip()
print(tString + string)
```
Explanation:

* `sys.stdin.readline()` reads a line from the standard input.
* `strip()` removes any leading or trailing whitespace from the line.
* `print()` prints the concatenated string to the standard output.

Note: The `strip()` method is used to remove any leading or trailing whitespace from the input strings, as the Java code in the question does not do this.</s>
